aten/src/ATen/native/native_functions.yaml

```diff
@@ -6347,7 +6347,7 @@
   device_check: NoCheck   # TensorIterator
   variants: method, function
   dispatch:
-    CPU, CUDA: zero_
+    CPU, CUDA, QuantizedCPU: zero_
```

Comment:
@albanD do you know why this might be needed? As I commented on PR Zeroing out a float tensor is not the same as filling quantized tensor with just zeros.

Reply:
I don't know why this is needed no. Since there is no testing in this PR, I doubt any new codepath is actually exercised so I suspect this is not needed.
If you have specific test that requires this, I'm sure we can figure out why it is needed if it is indeed.